### ISSUE TITLE
temprtaure module doesnt work

### ISSUE THREAD

**Issue by paradox-sp** [bcyran/fancy-motd#19]
sensors command shows the temperatures but motd doesnt show the temperature

**Comment by paradox-sp** [comment]
Fixed this in my fork [https://github.com/paradox-sp/fancy-motd](url)

**Comment by bcyran** [comment]
Unfortunately, your version does not work correctly on my machine, whereas mine does.

Your version:
![2023-11-15_19:41:50](https://github.com/bcyran/fancy-motd/assets/8322846/1b10f7f0-c818-4c76-8cc4-0388605a5a2f)

My version:
![2023-11-15_19:41:38](https://github.com/bcyran/fancy-motd/assets/8322846/e3e50528-8b02-4fc5-bfea-3de8ce7a95e6)

For reference, this is my sensors version:
```
❯ sensors -v
sensors version 3.6.0+git with libsensors version 3.6.0+git
```

And this is relevant sensors output on my machine:
```
Core 0:        +40.0°C  (high = +100.0°C, crit = +100.0°C)
Core 4:        +41.0°C  (high = +100.0°C, crit = +100.0°C)
Core 8:        +38.0°C  (high = +100.0°C, crit = +100.0°C)
Core 12:       +41.0°C  (high = +100.0°C, crit = +100.0°C)
Core 16:       +38.0°C  (high = +100.0°C, crit = +100.0°C)
Core 20:       +40.0°C  (high = +100.0°C, crit = +100.0°C)
Core 24:       +38.0°C  (high = +100.0°C, crit = +100.0°C)
Core 25:       +38.0°C  (high = +100.0°C, crit = +100.0°C)
Core 26:       +38.0°C  (high = +100.0°C, crit = +100.0°C)
Core 27:       +38.0°C  (high = +100.0°C, crit = +100.0°C)
Core 28:       +43.0°C  (high = +100.0°C, crit = +100.0°C)
Core 29:       +43.0°C  (high = +100.0°C, crit = +100.0°C)
Core 30:       +43.0°C  (high = +100.0°C, crit = +100.0°C)
Core 31:       +43.0°C  (high = +100.0°C, crit = +100.0°C)
```

It's probably a matter of differing output between sensors versions. It would be probably possible to figure out a code to handle both of them. It would be helpful if you posted your version and how the output looks on your machine.

Maybe I'll look into it in a spare time, but I don't promise anything. PRs always welcome :wink:

**Comment by quorn23** [comment]
Mine didn't work either. 
Debian box.
`sensors version 3.6.0 with libsensors version 3.6.0`

If i try the sensors grep line on the cli i get an issue with the symbols not showing up, running as motd the temp isn't showing up at all.

![Tabby_AL4JtfbPuX](https://github.com/bcyran/fancy-motd/assets/6703012/2b2f66b7-048a-475c-8f6e-99017482c2ff)

I fixed it with:
`cores="$(sensors | grep Core | awk '{printf "%s ", $3} {printf "%s %s %s\n", $6, $9, $12}' | tr -d '+°,' | sed 's/)//')"`

![Tabby_wVHaAp4aJl](https://github.com/bcyran/fancy-motd/assets/6703012/d81f4ebd-8d61-4189-9948-e3753afd041b)

If this warrants a PR let me know and i'll do one :)

**Comment by jackra1n** [comment]
I'm havin the same issue

**Comment by jackra1n** [comment]
in my case i had to do: `cores="$(sensors | grep Tctl | awk '{printf "%s ", $2}' | tr -d '+°,' | sed 's/)//')"`

**Comment by bcyran** [comment]
@quorn23 Fixed in 19b515c25c60b68a3b60f311e76b3e4484d3ffd2
@jackra1n Fixed in e3d3dff71630c38fb117ddac05155d7984d8d4c9
